lectures/getting_started_julia/julia_essentials.md

```diff
@@ -968,7 +970,7 @@ function solvemodel(x)
     a = x^2
     b = 2 * a
     c = a + b
-    return (;a, b, c)  # note local scope of tuples!
+    return (; a, b, c)  # note local scope of tuples!
 end
 
 solvemodel(0.1)
```

Comment:
Can we rename this function to `solve_model` instead so it is more consistent with notation.